lib/auth/accountrecovery.go

```diff
@@ -289,13 +232,12 @@ func (a *Server) VerifyAccountRecovery(ctx context.Context, req *proto.VerifyAcc
 	return approvedToken, nil
 }
 
-// verifyAuthnWithRecoveryLock counts number of failed attempts at providing a valid password or second factor.
-// After MaxAccountRecoveryAttempts, user's account is temporarily locked from logging in, recovery attempts are reset,
-// and all user's tokens are deleted. Modeled after existing function WithUserLock.
-func (a *Server) verifyAuthnWithRecoveryLock(ctx context.Context, startToken types.UserToken, authenticateFn func() error) error {
+// verifyAuthnRecoveryWithRecord validates the recovery code (through authenticateFn).  If the code is invalid,
+// a failed recovery attempt will be recorded.
+func (a *Server) verifyAuthnRecoveryWithRecord(ctx context.Context, startToken types.UserToken, authenticateFn func() error) error {
```

Comment:
```suggestion
func (a *Server) verifyAuthnRecovery(ctx context.Context, startToken types.UserToken, authenticateFn func() error) error {
```

Similar to the previous change, we can get rid of the `WithRecord` suffix.

Reply:
Updated, thank you